public/index.html

```diff
@@ -36,6 +36,18 @@
 	
 </head>
 
-<body>
+<body class="dark-mode">
 </body>
+<script>
+	function toggleTheme() {
+		if(document.body.classList.value == 'dark-mode') {
```

Comment:
``document.body.classList`` returns an array. It would be better if it contains ``dark-mode`` to be able to add more class to the body element later.